src/test/scala/uk/gov/hmrc/apiplatform/modules/apis/domain/models/ApiDefinitionFactory.scala

```diff
@@ -39,4 +49,19 @@ trait ApiDefinitionFactory {
       List.empty
     )
   }
+
+  def buildExtendedDefinition(versions: List[ExtendedApiVersion]): ExtendedApiDefinition = {
+    ExtendedApiDefinition(
+      ServiceName("test1ServiceName"),
+      "someUrl",
+      "test1Name",
+      "test1Desc",
+      ApiContext("som/context/here"),
+      versions = versions,
+      requiresTrust = false,
+      isTestSupport = false,
+      None,
```

Comment:
It helps with readability to name the arguments when the values are `None` and `List.empty`, like in the preceding lines with `false`.